x-pack/platform/plugins/shared/fleet/public/components/uninstall_command_flyout/uninstall_command_flyout.test.tsx

```diff
@@ -159,7 +160,9 @@ describe('UninstallCommandFlyout', () => {
     it('when user selects Windows, it renders commands for Windows', () => {
       const renderResult = render();
 
-      renderResult.getByTestId('windows').click();
+      act(() => {
+        renderResult.getByTestId('windows').click();
```

Comment:
shouldn't we use `userEvent` here too?

Reply:
`userEvent` can be used too. I went back and forth between `userEvent` and `act`, depending on which I thought was simpler for that particular test. We can update it if you'd like. You can either do it yourself or just let me know 🙏

Reply:
I pushed an update to that file 👍